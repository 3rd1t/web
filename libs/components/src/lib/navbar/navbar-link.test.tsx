import React from "react"
import { render } from "@testing-library/react"
import renderer from "react-test-renderer"
import NavbarLink from "./navbar-link"

describe(" NavbarLink", () => {
  it("should render successfully", () => {
    const { baseElement } = render(<NavbarLink label="Product" />)
    expect(baseElement).toBeTruthy()
  })
  it("renders correctly", () => {
    const tree = renderer.create(<NavbarLink label="Product" />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
