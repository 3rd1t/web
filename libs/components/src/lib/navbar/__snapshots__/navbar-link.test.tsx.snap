// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` NavbarLink renders correctly 1`] = `
<a
  className="text-lg font-medium text-gray-700 transition duration-150 ease-in-out hover:text-black"
  href="/"
  onClick={[Function]}
  onMouseEnter={[Function]}
>
  Product
</a>
`;
